Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.py

```diff
@@ -232,28 +246,47 @@ def edl_update():
             if verbose:
                 md = tableToMarkdown('List items:', list_items, headers=[list_name])
             else:
-                md = 'Instance context updated successfully'
+                md = 'Instance context updated successfully.'
 
     demisto.setIntegrationContext(dict_of_lists)
+
     demisto.results({
         'ContentsFormat': formats['markdown'],
         'Type': entryTypes['note'],
         'Contents': md
     })
 
-    # scp internal list to file_path
-    result = edl_update_external_file(file_path, list_name, verbose)
-    if result:
-        if verbose:
-            md = tableToMarkdown('Updated File Data:', result, headers=['Data'])
-        else:
-            md = 'External file updated successfully'
 
-        demisto.results({
-            'Type': entryTypes['note'],
-            'Contents': md,
-            'ContentsFormat': formats['markdown']
-        })
+def edl_update_internal_list_command():
+    """
+        Updates the instance context with the list name and items given
+    """
+    list_name = demisto.args().get('list_name')
+    list_items = argToList(demisto.args().get('list_items'))
+    add = demisto.args().get('add_or_remove') == 'add'
+    verbose = demisto.args().get('verbose') == 'true'
+
+    edl_update_internal_list(list_name, list_items, add, verbose)
+
+
+def edl_update():
+    """
+    Updates the instance context with the list name and items given
+    Overrides external file path with internal list
+    """
+    file_path = demisto.args().get('file_path')
+    if DOCUMENT_ROOT:
+        file_path = os.path.join(DOCUMENT_ROOT, file_path)
+    list_name = demisto.args().get('list_name')
+    list_items = argToList(demisto.args().get('list_items'))
+    add = demisto.args().get('add_or_remove') == 'add'
```

Comment:
please check that you don't get anything else than "add" or "remove"
I you don't want to remove if you got a bad value.